index.html

```diff
@@ -467,27 +478,25 @@
     var minute = (now - schedule[index].start)/60000;
     minute = minute/getPeriodLength(index);
     minute = minute * 2 * Math.PI;
-    drawHand(ctx, minute, radius*0.83, radius*0.05, 'silver');
+    drawHand(ctx, minute, radius*0.83, radius*0.05, 'silver', 0, 0);
 
     // second
     second=(second*Math.PI/30);
-    drawHand(ctx, second, radius*0.9, radius*0.02, 'red');
+    drawHand(ctx, second, radius*0.9, radius*0.02, 'red', 0, 0);
 }
 
 // ctx : canvas context
 // pos : angle out of 360, 90 is east
 // length, width : self explanatory
-// color : CSS color ('#FF0000' is red)
-function drawHand(ctx, pos, length, width, color) {
+// color : CSS color ('#FF0000' is red) 
+function drawHand(ctx, pos, length, width, color, x, y) {

```

Comment:
The x and y parameters should be described in the comment, like the other parameters. I assume they're for the start of the hand at the center of the face, but it can't hurt to specify.